releasenotes/notes/single-meta-in-azureconverter-ce1cc196a9b161f3.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Adds support for single metadata dictionary input in `AzureOCRDocumentConverter`.
```

Comment:
Can you explain better what this feature is about? The PR description is already a good example

Reply:
```suggestion
    Adds support for single metadata dictionary input in `AzureOCRDocumentConverter`. In this way, additional metadata can be added to all files processed by this component even when the length of the list of sources is unknown.

```